### PR TITLE
Add Disposal Test and fix an issue where the CopyClipboard test was failing

### PR DESCRIPTION
**Pull request by a-usr**
Fixes #???? - Include a terse summary of the change or which issue is fixed.

## Pull Request checklist:

- [ ] I've named my PR in the form of "Fixes #issue. Terse description."
- [x] My code follows the [style guidelines of Terminal.Gui](https://github.com/gui-cs/Terminal.Gui/blob/develop/.editorconfig) - if you use Visual Studio, hit `CTRL-K-D` to automatically reformat your files before committing.
- [x] My code follows the [Terminal.Gui library design guidelines](https://github.com/gui-cs/Terminal.Gui/blob/develop/CONTRIBUTING.md)
- [x] I ran `dotnet test` before commit
- [ ] I have made corresponding changes to the API documentation (using `///` style comments)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any poor grammar or misspellings
- [x] I conducted basic QA to assure all features are working

### CODE REVIEW

**Comment by a-usr** [comment]
please?

File: UnitTests/Views/ViewDisposalTest.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		}
+#nullable enable
+		Dictionary<Type, object? []?> special_params = new Dictionary<Type, object? []?> ();
+#nullable restore
+		[Fact]
+		[AutoInitShutdown]
+		public void TestViewsDisposeCorrectly ()
+		{
+			var reference = DoTest ();
+			for (var i = 0; i < 10 && reference.IsAlive; i++) {
```
Comment (tig) [inline UnitTests/Views/ViewDisposalTest.cs:32]: Thank you for showing how to do this! I have been wondering if there's a way to poke into GC to see if things were really released or not.

File: UnitTests/Views/ViewDisposalTest.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+			Toplevel top = Application.Top;
+			var views = GetViews ();
+			foreach (var view in views) {
+				View instance;
+				//Create instance of view and add to container
+				if (special_params.ContainsKey (view)) {
+					instance = (View)Activator.CreateInstance (view, special_params [view]);
+				}
+				else
+					instance = (View)Activator.CreateInstance (view);
```
Comment (tig) [inline UnitTests/Views/ViewDisposalTest.cs:60]: Please add `{` and `}` (we try to be very consistent about always using braces for nested statements). Thanks.

File: UnitTests/Views/ViewDisposalTest.cs [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+namespace UnitTests.ViewsTests {
+	public class ViewDisposalTest {
+		ITestOutputHelper output;
+		public ViewDisposalTest (ITestOutputHelper output)
+		{
+			{
+				this.output = output;
+			}
+		}
+#nullable enable
```
Comment (tig) [inline UnitTests/Views/ViewDisposalTest.cs:24]: Please add vertical whitespace between members.

File: UnitTests/Views/ViewDisposalTest.cs [hunk 4]
```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```
Comment (tig) [inline UnitTests/Views/ViewDisposalTest.cs:3]: Are all these `using` required? Please remove any that are not needed.

**Review by tig — COMMENTED** [review]
Lovely! I'll merge as soon as you address my comments.

**Review by tig — APPROVED** [review]

**Pull request merged**